pilot/pkg/serviceregistry/kube/controller/ambientindex.go

```diff
@@ -854,8 +868,13 @@ func (a *AmbientIndex) handleService(obj any, isDelete bool, c *Controller) sets
 					Namespace: svc.Namespace,
 					Domain:    "cluster.local",
 				})),
-				Address: svcIP.AsSlice(),
-				Ports:   ports,
+				Addresses: []*workloadapi.NetworkAddress{
+					{
+						Address: svcIP.AsSlice(),
+						Network: c.Cluster().String(),
```

Comment:
I network same as cluster ? I thought they are separate ( a cluster may have workload entries in different network ). 

I do think we should include the cluster too.

Reply:
was a typo, using the controller.Network() function for correctly getting network value